### PR TITLE
Add public API

### PR DESCRIPTION
**Pull request by ksh8281**

### CODE REVIEW

File: src/api/EscargotPublic.h [hunk 1]
```diff
@@ -1656,7 +1656,7 @@ typedef TemplateNamedPropertyHandlerEnumerationCallbackResultVector (*TemplateNa
 // if intercepted you may returns non-empty value.
 // the returned value will be use futuer operation(you can return true, or false)
 typedef OptionalRef<ValueRef> (*TemplateNamedPropertyHandlerDefineOwnPropertyCallback)(ExecutionStateRef* state, ObjectRef* self, void* data, const TemplatePropertyNameRef& propertyName, const ObjectPropertyDescriptorRef& desc);
-typedef OptionalRef<ObjectRef> (*TemplateNamedPropertyHandlerGetPropertyDescriptorCallback)(ExecutionStateRef* state, ObjectRef* self, void* data);
+typedef OptionalRef<ValueRef> (*TemplateNamedPropertyHandlerGetPropertyDescriptorCallback)(ExecutionStateRef* state, ObjectRef* self, void* data, const TemplatePropertyNameRef& propertyName);
```
Comment (bbrto21) [inline src/api/EscargotPublic.h:1659]: I think that you have to add a test case for this.

Comment (ksh8281) [reply src/api/EscargotPublic.h:1659]: You are right. I added test case in testcc

**Review by clover2123 — APPROVED** [review]
LGTM

**Review by bbrto21 — APPROVED** [review]
LGTM

**Pull request merged**